adapter/internal/adapter/adapter.go

```diff
@@ -188,7 +183,13 @@ func Run(conf *config.Config) {
 	// Set enforcer startup configs
 	xds.UpdateEnforcerConfig(conf)
 
-	go operator.InitOperator()
+	go operator.InitOperator(conf.Adapter.Metrics.Port)
+
+	// Start the metrics server
+	if conf.Adapter.Metrics.Enabled && strings.EqualFold(conf.Adapter.Metrics.Type, metrics.PrometheusMetricType) {
```

Comment:
I hope we have parameterized it in the values.yaml. prometheus enable/disable, and configs under components

Reply:
Yes, the metrics enabling/disabling is done using the values.yaml file, and the relevant configurations have been added to the ConfigMaps and the default config of both the adapter and common controller.